### DISCUSSION TITLE
IESG state help pages

### DISCUSSION THREAD

**Discussion by cindymorgan** [ietf-tools/datatracker#3794] [Ideas]
From John Scudder:

> Each datatracker document page (for example, https://datatracker.ietf.org/doc/draft-ietf-detnet-bounded-latency/) has a clickable link, “IESG state”, which points to https://datatracker.ietf.org/help/state/draft/iesg. If instead, one clicks through on the “Edit” button, one sees a “Help on states” button that links to https://datatracker.ietf.org/doc/help/state/draft-iesg/
> 
> The two help pages are redundant, but the second one is better because it includes the very handy state diagram (“View diagram”).
> 
> Can we get rid of https://datatracker.ietf.org/help/state/draft/iesg completely, and retarget any references to it, to point to https://datatracker.ietf.org/doc/help/state/draft-iesg/ ? Or, turn it into a redirect, similar?
> 
> Thanks,

**Comment by rjsparks** [comment]
Yeah there's a lot of duplication simplification we can pursue here.
At /help/state, there is this:
![image](https://user-images.githubusercontent.com/10996692/162502086-d209459f-83d8-4920-8c56-e69dbbe28309.png)
All of the links in that top row could be served by the views at /doc/help/state rather than at /help/state.

It's actually better to keep the doc specific state descriptions under /doc/help since those states are defined in ietf.doc.models rather than all the other states, which are in ietf.name.models.
